Assets/Scripts/UI/GameCursor.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using UnityEngine;
+
+public class GameCursor : MonoBehaviour
+{
+    private const string CameraModeKey = "cameraType";
+
+    private static CameraMode _CameraMode = CameraMode.Original;
+    private static CursorMode _CursorMode = CursorMode.Menu;
+
+    public static CameraMode CameraMode
+    {
+        get { return _CameraMode; }
+        set { SetCameraMode(value); }
+    }
+
+    public static CursorMode CursorMode
+    {
+        get { return _CursorMode; }
+        set { SetCursorMode(value); }
+    }
+
+    public static void ApplyCameraMode()
+    {
+        SetCameraMode(CameraMode);
+    }
+
+    public static void ApplyCursorMode()
+    {
+        SetCursorMode(CursorMode);
+    }
+
+    public static void Cycle()
+    {
+        switch (CameraMode)
+        {
+            case CameraMode.Original:
+                CameraMode = CameraMode.WOW;
+                break;
+
+            case CameraMode.WOW:
+                CameraMode = CameraMode.TPS;
+                break;
+
+            case CameraMode.TPS:
+                CameraMode = CameraMode.Original;
+                break;
+
+            default:
+                Debug.LogWarning($"Tried to cycle in {CameraMode} mode");
+                return;
+        }
+    }
+
+    private static void LoadPlayerPrefs()
+    {
+        var savedValue = PlayerPrefs.GetInt(CameraModeKey, (int)CameraMode);
+        if (Enum.IsDefined(typeof(CameraMode), savedValue))
```

Comment:
I added this, because I had changed the type of enum that was stored in PlayerPrefs... So the old value was no longer defined by the new enum.

Reply:
So if this only occurs with an earlier version of 164, then we don't need this code within our branch

Reply:
> So if this only occurs with an earlier version of 164, then we don't need this code within our branch

It's fixing a potentially breaking change should we ever change the enum. Let's keep it; it doesn't do any harm.